app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -6,19 +6,41 @@ body: |
   that would be useful to them. This component would be used to add this personalised content and would
   indicate to the user that this is not normally part of the page, but has been added for them specifically.
 
-  Right now the contents of the component are static, as the MVP of personalised content is only for Start a Business.
-  Since many pages will use this component with the same text, we hard-code it here for now.
+  Parameters:
+
+    - `suggestion_text`: the text to display first, before the link
+    - `suggestion_link_text`: the text of the link
+    - `suggestion_link_text`: the URL of the link
+    - `dismiss_text`: the text of the link used to dismiss the intervention
+
+  The dismiss link will reload the page but the `hide-intervention` query string parameter will cause the back-end not to show the intervention again. Some progressive enhancement will be added in later to avoid reloading the page if JavaScript is available.
 
-  The dismiss link will reload the page but the `hide-intervention` query string parameter will cause the
-  backed not to show the intervention again. Some progressive enhancement will be added in later to avoid
-  reloading the page if JavaScript is available.
 accessibility_criteria: |
   The intervention component must:
 
   - have a border colour contrast ratio of more than 4.5:1 with its background to be visually distinct
+  - be usable using the keyboard
+  - make the focused links/controls visible
```

Comment:
I'm not sure what this means. Is the accessibility of the links not already covered by the shared accessibility criteria for a link?